bedrock/firefox/templates/firefox/australis/growth-firstrun-test2.html

```diff
@@ -0,0 +1,163 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/australis/tour-base.html" %}
+
+{% set_lang_files "firefox/australis/firefox_tour" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{ _('Welcome to Firefox') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block body_id %}firstrun{% endblock %}
+
+{% block telemetry_id %}australis-tour-firstrun-35-test2{% endblock %}
+
+{% block doorhanger_text %}
+  data-text="{{ _('Get more from Firefox by learning one new thing.') }}"
+{% endblock %}
+
+{% block doorhanger_title %}
+  data-title="{{ _('Learn something new!') }}"
+{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'growth_firstrun_test2' %}
+{% endblock %}
+
+{% block js %}
+  {% javascript 'growth_firstrun_test2' %}
+{% endblock %}
+
+{% block main_heading %}
+  <header role="banner">
+    <div class="logo"></div>
+    <div class="stage">
+      <h1>{{ _('Welcome to Firefox') }}</h1>
+      <p>{{ _('Committed to you, your privacy and an open Web.') }}</p>
+    </div>
+  </header>
+{% endblock %}
+
+{% block tour_content %}
+  <div class="tour-init" data-target="appMenu" data-icon="{{ MEDIA_URL }}img/firefox/australis/logo.png?2014-03" data-icon-high-res="{{ MEDIA_URL }}img/firefox/australis/logo-high-res.png?2014-03"></div>
```

Comment:
Late night rebase troubles meant I forgot these were supposed to change. I just needed it to work. :smile: Updating.